python/ray/ml/train/integrations/huggingface/huggingface_trainer.py

```diff
@@ -236,6 +237,17 @@ def trainer_init_per_worker(train_dataset, eval_dataset, **config):
 
     _checkpoint_manager_cls = _DataParallelSyncingCheckpointManager
 
+    _dataset_config = {
+        "train": DatasetConfig(
+            fit=True,
+            split=True,
+            required=True,
+            streamable=False,
+            _noncustomizable_fields=["streamable"],
+        ),
+        "evaluation": DatasetConfig(),
+    }
```

Comment:
cc @Yard1 to confirm if this default configuration is correct

Reply:
Does split=True means we will split all datasets? We should only split the train dataset here. Aside from that, looks good. We could add streaming support later, but right now, it probably won't work.

Reply:
Gotcha, I see. Looks good then!